pkg/controllers/trusted_ca_bundle_controller.go

```diff
@@ -0,0 +1,212 @@
+package controllers
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/util"
+	"io/ioutil"
```

Comment:
There's currently a mix of imports here, core imports should be separated from external packages, please gofmt this file

```suggestion
	"io/ioutil"
	
	configv1 "github.com/openshift/api/config/v1"
	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/util"
```